Test arbitrary moose-environment choice

milljm opened the pull request:
This hash represents the _previous_ moose-environment package (PR-169), thus


Closes #12559

Commit:
Test arbitrary moose-environment choice

This hash represents the _previous_ moose-environment package (PR-169), thus
DO NOT MERGE ME.

Closes #12559

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/262476/) on 4d16464 wanted to post the following:

View the site [here](http://mooseframework.inl.gov/docs/PRs/12560/site)

This comment will be updated on new commits.

Commit:
Tag moose-environment package version

MOOSE will use https://github.com/idaholab/package_builder/pull/175
as the default moose-environment

Closes #12559

Commit:
Allow some additional information to be stored

Allow comments, etc to the package_version file.

Commit:
Add the PR reference in which package is based

rwcarlsen commented:
Are CIVET changes ready that go with this?

friedmud reviewed (approved):
👍
